source/fundamentals/monitoring.txt

```diff
@@ -0,0 +1,406 @@
+==========
+Monitoring
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+.. What is on this page?
+
+This guide explains how you can set up and configure **monitoring** in the
+MongoDB Java driver.
+
+.. What do any new terms mean?
+
+Monitoring is the process of getting information about the activities a piece of
+software performs. You can use the information you get from monitoring in your
+application or an application performance management library. 
+
+Monitoring the MongoDB Java driver lets you understand the
+driver's resource usage and performance, and can help you make informed
+decisions when designing and debugging your application.    
+
+.. What can you expect to see on this page?
+
+On this page you will learn how to perform these tasks:
+
+- :ref:`Monitor different types of events in the MongoDB Java Driver <monitoring-monitor-events>`
+- :ref:`Monitor connection pool events with Java Management Extensions (JMX) and JConsole <monitoring-jmx>`
+
+This page shows how to use information about the activity of the driver in code.
+If you would like to learn how to record events the driver performs,
+consider reading our :doc:`guide on logging </fundamentals/logging>`.
+
+.. _monitoring-monitor-events:
+
+Monitor Events
+--------------
+
+To monitor an **event**, you must register a **listener** on your ``MongoClient``
+instance. 
+
+An event is any action that happens in code. The driver includes functionality
+for listening to a subset of the events that occur when the driver is running.
+
+A listener is a class that performs some action when certain events occur. All
+listener classes extend the ``EventListener`` interface. A listener's
+API defines the events it can respond to.
+
+Each method of a listener class represents a response to a certain event. Each
+method receives one argument: an object representing the event the method
+responds to. All event objects are a subclass of the ``EventObject`` class.  
+
+The MongoDB Java driver organizes the events it defines into three categories:
+
+- Command Events
```

Comment:
I agree that this could be helpful for some readers, but I believe that turning these items into links would[ not provide sufficient context](https://docs.mongodb.com/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-text).

> Begin a cross-reference sentence with the purpose or benefit of the cross-reference (such as more information or examples). Such context helps users decide whether to follow the reference.

I do not wish to rephrase this sentence to be about the page itself (I want this sentence to be focused on the driver rather than the page), so I would like to keep this as it is currently written (not linking).

Reply:
Since it's only 3 items and none are being defined here, can it be in a sentence? This would save vertical space.

Reply:
The style guide wants lists of three or more items broken out of paragraphs into lists.

[Section of Style Guide](https://docs.mongodb.com/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#create-short-paragraphs)

>  When listing three or more items, use a bullet list instead of embedding the items in a paragraph.

I also personally prefer this style, as I think it is easier on the eyes.

Based off of this section of the style guide, I think it is correct as written.